pkgs/development/tools/misc/k2tf/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "k2tf";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "sl1pm4t";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0i1bhn0sccvnqbd4kv2xgng5r68adhcc61im2mn8hxmds5nf6in2";
+  };
+
+  vendorSha256 = "1c2mwhrj0xapc661z1nb6am4qq3rd1pvbvjaxikjyx95n0gs8gjk";
+
```

Comment:
Could you please check if you could set the version?

```bash
buildFlagsArray+=("-ldflags" "-s" "-w" "-X whatever=v${version}")
```

Reply:
followup in https://github.com/NixOS/nixpkgs/pull/114368